velox/vector/arrow/Bridge.cpp

```diff
@@ -38,10 +38,17 @@ static constexpr size_t kMaxBuffers{3};
 // carried by ArrowArray.private_data
 class VeloxToArrowBridgeHolder {
  public:
-  VeloxToArrowBridgeHolder() {
-    for (size_t i = 0; i < kMaxBuffers; ++i) {
+  VeloxToArrowBridgeHolder() = default;
+
+  void resizeBuffers(size_t bufferCount) {
+    if (bufferCount <= numBuffers_)
```

Comment:
please always use bracket (even for single line) - check https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

Reply:
Done